api/execution/gen.go

```diff
@@ -1,3 +1,3 @@
 package execution
 
-//go:generate protoc -I.:../..:../../../../../github.com/gogo/protobuf:/usr/local/include --gogoctrd_out=plugins=grpc,import_path=github.com/docker/containerd/api/execution,Mgogoproto/gogo.proto=github.com/gogo/protobuf/gogoproto,Mgoogle/protobuf/descriptor.proto=github.com/gogo/protobuf/protoc-gen-gogo/descriptor:. execution.proto
+//go:generate protoc -I.:${PWD}/vendor/github.com/gogo/protobuf:/usr/local/include --gogoctrd_out=plugins=grpc,import_path=github.com/docker/containerd/api/execution,Mgogoproto/gogo.proto=github.com/gogo/protobuf/gogoproto,Mgoogle/protobuf/descriptor.proto=github.com/gogo/protobuf/protoc-gen-gogo/descriptor:. execution.proto
```

Comment:
@stevvooe just FYI, it does appear via the `Makefile` or with a bare `go generate` on the command line, `${PWD}` is being evaluated in both cases. However, it may not matter that much as there will still be lots of `../` paths even with `${PWD}` available. So I will update with the path set from above and hopefully that will get this PR ready to move forward with.